controllers/suite_test.go

```diff
@@ -62,7 +62,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
+	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(false)))
```

Comment:
I turned debug mode off in the tests because is there is so much logging that is practically impossible to follow. It's better turn it on just if needed to debug some test.